### PR TITLE
Vulparace

### PR DESCRIPTION
**Pull request by Poojawa**
Vulpkanin race from Paradise!

Canine/vulpine base that isn't really old sprites from Citadel Station.

Facial hair allows you to change cheek floof as a sort of poor man's second color layer. Plans are in place to add an 'under belly' and 'gloves' to each sprite so proper fox color ratios can be maintained. 

Tail has wagging abilities, ~~but they're disabled pending conversion to Polaris' formats~~ Should be fixed. <3

Canine speech, made of suitably canine noises. Rfft.

Lesser form is Wolpin, but no cubes added for approval.

Custom hardsuit/masks/helmet support, they're actually pretty bad ass tbh. RIG sprites included.

### CODE REVIEW

**Comment by Poojawa** [comment]
Hooray!

**Comment by Arokha** [comment]
We aren't changing any stock Polaris .dmi files, ever, and only changing stock .dm files in very limited ways. Like `icons/mob/human_face.dmi` can't change. Neither can `icons/mob/human_races/monkeys/tail.dmi`. Look at how Sergal tails are done in station_vr.dm, you can set their tail icon to a different file, or use the icobase_tail variable and put their tail in their r_species.dmi file named 'tail' (or is it tail_s? Look in r_sergal.dmi).

Overall, to get merged:
`species/station/station.dm` readd the linebreak at the end so this file doesn't change.
`species/station/monkey.dm` should go in `monkey_vr.dm`
`language/station.dm` needs to be in `station_vr.dm`, look for Sagaru and the others we added.
`stylesheet.dm` change is unavoidable, this is fine.
`sprice_accessories_vr.dm` is good.
`species/station/station_vr.dm` is good.
All the other icon files are fine since they are new and don't conflict with Polaris.

**Comment by Cameron653** [comment]
If you want the voidsuit/rigsuit sprites to work, you're going to need to define it in
`/code/modules/clothing/spacesuits/rig/rig_pieces_vr.dm` (So they get the correct rig sprite)
`/code/game/machinery/suit_storage_unit_vr.dm `(So the suit cycler has that specific race)
`/code/modules/clothing/spacesuits/void/void_vr.dm` (Correct spritesheet for voidsuits.)

**Comment by SpadesNeil** [comment]
The whole _vr is to make sure the vore station code stays organized and distinguishable from Polaris in case we ever need to port things again, and so there's no conflicts with the code upstream so we can port all the cool things they often fix.

If you just change this to be consistent with how we want our code written, you're good.

**Comment by Poojawa** [comment]
Ah, hah. Will do.

**Comment by Poojawa** [comment]
Changes applied. all polaris vanilla files were copy/pasta'd from raw views of the master copies for Virgo code. so any alterations can safely be disregarded for any potential conflicts.

**Comment by killer653** [comment]
Still need to add them to the suit cycler as listed above so they can get void suits to work.

**Comment by Poojawa** [comment]
ohw hoops, I didn't even look into the suit locker.

**Comment by Arokha** [comment]
```
code/modules/mob/language/station.dm 
code/modules/mob/living/carbon/human/species/station/monkey.dm 
code/modules/mob/living/carbon/human/species/station/station.dm 
```

These 3 files remain to be reverted to their original Polaris form. After that, I can remove the DNM tag. You can download current copies from master and put those in a commit on your branch, or you can do different stuff depending on the client you're using to download a copy pre-changes.

`species/station/station.dm` in particular just needs a linebreak at the end it looks like.

**Comment by Poojawa** [comment]
Err, They're literally copy/pasta'd of master copies, there shouldn't be any issue in any of them. o.o

**Comment by SpadesNeil** [comment]
I think this is all set. I'll assign Arokha to review and merge it if they don't see problems. I don't.

**Comment by SpadesNeil** [comment]
Also, @Poojawa, be sure to include info about this new race on the Backstory page of our wiki!

https://wiki.vore-station.net/Backstory

**Comment by Arokha** [comment]
Tested a bit, seems good.

One last change: Can you remove their language being default for them? We've sort of undone that on all the others so if you wanna play X but actually use the custom_species setting to be a Y with the body style of X, then you can know whatever languages are appropriate. Like you can pick 2 instead of pick 1 + the default language for your body style.

Just comment that line out I guess.

Also who did the tail animations?

**Comment by Poojawa** [comment]
I pretty much just copied and pasted the same animation to meet the minimum number, as just having one made it invisible nine times out of ten.

As for the language change, easy enough~!

**Comment by SpadesNeil** [comment]
This looks good to be added. @Poojawa just make a Wiki entry and we'll merge this. It's required for new races.

**Comment by Poojawa** [comment]
I'll try. lol.

**Comment by Poojawa** [comment]
@SpadesNeil Backstory written, as much as I could flex before work \o/

**Comment by SpadesNeil** [comment]
I seem to have created a conflict that must be resolved first, probably having to do with the thing I just ported in from Leshana fixing suits.

File: code/modules/clothing/spacesuits/void/void_vr.dm [hunk 1]
```diff
@@ -7,9 +7,12 @@
 		"Akula" 				= 'icons/mob/species/akula/helmet_vr.dmi',
 		"Sergal"				= 'icons/mob/species/sergal/helmet_vr.dmi',
 		"Flatland Zorren" 		= 'icons/mob/species/fennec/helmet_vr.dmi',
-		"Highlander Zorren" 	= 'icons/mob/species/fox/helmet_vr.dmi'
+		"Highlander Zorren" 	= 'icons/mob/species/fox/helmet_vr.dmi',
+		"Vulpkanin"				= 'icons/mob/species/vulpkanin/helmet.dmi'
 		)
 
+	sprite_sheets_refit = list(
```
Comment (Cameron653) [inline code/modules/clothing/spacesuits/void/void_vr.dm]: This is what is most likely causing the merge errors, due to #234

**Comment by Poojawa** [comment]
Yup, I'd made a comment about said merge conflicts in her PR, fixing~

**Comment by Poojawa** [comment]
Fiiiinally. 

and now I fuck of to bed.

**Comment by SpadesNeil** [comment]
This looks good to go. Perusing the code for issues.

File: code/modules/mob/language/station_vr.dm [hunk 2]
```diff
@@ -23,4 +23,18 @@
 	speech_verb = "snarls"
 	colour = "sergal"
 	key = "z"
-	syllables = list ("grr", "gah", "woof", "arf", "arra", "rah", "wor", "sarg")
\ No newline at end of file
+	syllables = list ("grr", "gah", "woof", "arf", "arra", "rah", "wor", "sarg")
+
+/datum/language/vulpkanin
+	name = "Canilunzt"
+	desc = "The guttural language spoken and utilized by the inhabitants of Vazzend system, composed of growls, barks, yaps, and heavy utilization of ears and tail movements.Vulpkanin speak this language with ease."
```
Comment (SpadesNeil) [inline code/modules/mob/language/station_vr.dm]: Minor grammar issue here.

"... utilization of ears and tail movements.Vulpkanin speak this ..."

There needs to be a space there.

**Comment by SpadesNeil** [comment]
That's all the problems I can find with it, and it's not stopping it from being merged by next month.

**Comment by Poojawa** [comment]
fixed!

**Pull request merged**